source/utilities.tex

```diff
@@ -15012,24 +15056,27 @@
 \item \ucode{20000} -- \ucode{2fffd}
 \item \ucode{30000} -- \ucode{3fffd}
 \end{itemize}
-The estimated width of other code points is 1.
+The field width of all other code points is 1.
 
 \pnum
-For a string in a non-Unicode encoding, the width of a string is unspecified.
+For a sequence of characters in a non-Unicode encoding,
+the field width is unspecified.
```

Comment:
Should `\unspec` be used here?

Reply:
no, \unspec is for library code (e.g. in a synopsis)